### ISSUE TITLE
Provide method to delete empty MISP Objects

### ISSUE THREAD

**Issue by github-germ** [MISP/MISP#5380]
This is spun out of https://github.com/MISP/PyMISP/issues/490 where we implemented a MISP Purge process to delete aged-out events and attributes. An issue of fully removing object was identified.

Currently, PyMISP does not offer any methods to access MISP Objects directly. A client can delete the attributes within an object. However, this leaves an empty object that cannot be deleted.

* Ideally, it might be nice to enhance PyMISP to include access to objects.
* At the very least, will it possible to get a method to remove these empty objects, e.g. via PyMISP, or perhaps via some instance level transaction similar to removing attribute orphans.

Thanks!

**Comment by iglocska** [comment]
/objests/delete/id on MISP side should handle this, @Rafiot - is this implemented on PyMISP side?

**Comment by Rafiot** [comment]
Yes, it is implemented. The problem is finding the empty objects without iterating through every event to search them.

**Comment by github-germ** [comment]
Is the deletion of empty objects still on the to-do list? Thanks.
